### PR TITLE
Add custom feedback icons

### PR DESCRIPTION
**Pull request by mrDinckleman**
Hi. I'm using FontAwesome Icons in my project, instead of Bootstrap Glyphicons.
My proposal is add ability to customize feedback icon classes, something like this.

### CODE REVIEW

**Comment by 1000hz** [comment]
I think I'd prefer to add just one option called `feedback` that's an object 
```js
{
  success: 'glyphicon-ok',
  error:   'glyphicon-warning-sign'
}
```

File: dist/validator.js [hunk 1]
```diff
@@ -215,9 +217,12 @@
   }
 
   Validator.prototype.defer = function ($el, callback) {
-    if (!this.options.delay) return callback()
+    var _this = this;
```
Comment (1000hz) [inline dist/validator.js]: Looks like a semi-colon snuck through

File: js/validator.js [hunk 2]
```diff
@@ -235,9 +237,12 @@
   }
 
   Validator.prototype.defer = function ($el, callback) {
-    if (!this.options.delay) return callback()
+    var _this = this;
```
Comment (1000hz) [inline js/validator.js]: Looks like a semi-colon snuck through

Comment (1000hz) [inline js/validator.js]: do `callback = $.proxy(callback, this)` instead of using apply and _this

**Comment by mrDinckleman** [comment]
Branch update

File: js/validator.js [hunk 3]
```diff
@@ -238,6 +242,7 @@
   }
 
   Validator.prototype.defer = function ($el, callback) {
+    callback = $.proxy(callback, this);
```
Comment (1000hz) [inline js/validator.js]: can you get rid of this semicolon?

**Comment by mrDinckleman** [comment]
Done

**Comment by 1000hz** [comment]
Thanks!

**Pull request merged**